eslint-plugin-expensify/tests/no-multiple-onyx-in-file.test.js

```diff
@@ -0,0 +1,40 @@
+const RuleTester = require('eslint').RuleTester;
+const rule = require('../no-multiple-onyx-in-file');
+const message = require('../CONST').MESSAGE.NO_MULTIPLE_ONYX_IN_FILE;
+
+const ruleTester = new RuleTester({
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module',
+    },
+});
+
+ruleTester.run('no-multiple-onyx-in-file', rule, {
+    valid: [
+        {
+            code: `
+                withOnyx({
+                    key: 'value',
+                })(Component);
+            `,
+        },
+    ],
+    invalid: [
+        {
+            code: `
+                withOnyx({
+                    key1: 'value1',
+                })(Component1);
+
+                withOnyx({
+                    key1: 'value1',
+                })(Component2);
+            `,
```

Comment:
I'm still not sure if the invalid code is correct. Should the test code be inside the `compose`?

Reply:
Ahh yes, thanks for pointing this out @mollfpr, the test case is already fixed and it's passed as expected
<img width="1552" alt="Screenshot 2023-09-19 at 22 48 57" src="https://github.com/Expensify/eslint-config-expensify/assets/20473526/a71bc841-3cbd-4a95-aaa9-383b42c588f0">